doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -962,6 +962,17 @@ HomeKit
 
 The issues in this section are related to the HomeKit protocol.
 
+.. rst-class:: v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+
+KRKNWK-17343: Accessories becoming significantly slower when some data pairs in the NVS change frequently.
+  Accessing data pairs that rarely change can take a long time.
+  This problem manifests itself for example, in case of long booting or paired/unpaired identify response time.
```

Comment:
```suggestion
  It might happen, for example, in case of long booting or paired/unpaired identify response time, which can lead to certification issues.
```